lib/bloc/crypto/crypto_bloc.dart

```diff
@@ -230,41 +240,54 @@ class CryptoBloc extends Bloc<CryptoEvent, CryptoState> {
           await apiCrypto.generateAccount(wallet, keyType, index);
       return account;
     } catch (e) {
-      print(e);
       rethrow;
     }
   }
 
   Future<Account> _initAccount(
       Wallet wallet, int index, KeyType keyType) async {
-    Account account = await _generateAccount(wallet, index, keyType);
-    account = await _syncAccount(account);
-    account = await _syncVtts(account);
-    await account.setBalance();
-    await db.addAccount(account);
-    return account;
+    try {
+      Account account = await _generateAccount(wallet, index, keyType);
+      account = await _syncAccount(account);
```

Comment:
This looks error-prone due to a possible `race condition`. As we are never using the account returned here, I suggest ignoring the result of this call to avoid errors in the future.

```suggestion
      await _syncAccount(account);
```